### ISSUE TITLE
Autocomplete plugin erase some text before a new tag if you have a new line

### ISSUE THREAD

**Issue by jona-1993** [ckeditor/ckeditor4#4921]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. …
2. …
3. …

### Expected result

*What is the expected result of the above steps?*

-------------------

Something, [Your tag]
-------------------

### Actual result

*What is the actual result of the above steps?*

-------------------
 
Some[Your tag]
-------------------

## Other details

* Browser: All browsers
* OS: Windows, Mac
* CKEditor version: 4
* Installed CKEditor plugins: 
autocomplete
base64image
basicstyles
button
ckeditor-gwf-plugin
clipboard
colorbutton
colordialog
computedfont
computedstyles
contextmenu
deselect
dialog
dialoggui
docfont
docprops
enterkey
entities
fakeobjects
floatingspace
floatpanel
font
googleDocPastePlugin
imagepaste
imageresize
imageresizerowandcolumn
imagerotate
ident
identlist
justify
list
listblock
magicline
menu
notification
pagebreak
panel
panelbutton
pastebase64
pastefromdocs
pastFromGoogleDoc
pastfromlibreoffice
pastefromword
pastetext
pastetools
richcombo
spacingsliders
table
tableresize
tabletools
tagsmedecins (My plugin for have my custom tags: the trigger is "#")
textmatch
textwatcher
toolbar
undo
wysiwygarea

**Comment by Comandeer** [comment]
I can't reproduce the issue using `autocomplete` and `textmatch` plugins: [https://jsfiddle.net/Comandeer/g2ydstvp/](https://jsfiddle.net/Comandeer/g2ydstvp/)

Could you provide more info about your `tagsmedecins` plugin?

**Comment by jona-1993** [comment]
Yes, the plugin tagmedecins is here: https://pastebin.com/6QpawvkR

I simplified, I have a lot of tags..

**Comment by Comandeer** [comment]
I still can't reproduce the issue with your custom plugin: [https://jsfiddle.net/Comandeer/g2ydstvp/2/](https://jsfiddle.net/Comandeer/g2ydstvp/2/)

Maybe some other plugin modifies content on insertion to the editor?
